pkg/ingester/ingester_v2.go

```diff
@@ -992,6 +992,11 @@ func (i *Ingester) v2Query(ctx context.Context, req *client.QueryRequest) (*clie
 
 	i.metrics.queries.Inc()
 
+	if i.State() == services.Stopping {
```

Comment:
I'm not sure this is actually a real change. `v2Query()` is called by `Ingester.Query()` which already does this check calling `i.checkRunningOrStopping()`.

_Same comment applies to other functions._

Reply:
Change is that while `i.checkRunningOrStopping()` is used to allow querying in `Stopping` state, here we disable it.

Reply:
Oh sorry. Well, then we should change the check in `Ingester.Query()`, no?

Reply:
That's an option too. Chunks code currently allows querying during Stopping, this change would break it. In chunks with transfers enabled, Stopping can take long time, so it makes sense there.

Reply:
Ok, let's keep it as is.

However, if we keep logging it, then maybe an utility similar to `checkRunningOrStopping()` may be a nice to have to avoid repeating the same log over and over.